src/components/map/MapService.js

```diff
@@ -972,13 +972,9 @@ goog.require('ga_urlutils_service');
           if (!this.isBodLayer(olLayer)) {
             return false;
           }
-          var hasTooltip;
           var parentLayerId = this.getBodParentLayerId(olLayer);
           var bodId = parentLayerId || olLayer.bodId;
-          if (bodId) {
-            hasTooltip = this.getLayerProperty(bodId, 'tooltip');
-          }
-          return !!(bodId && hasTooltip);
+          return this.getLayerProperty(bodId, 'tooltip');
```

Comment:
Yes but in that case that means the layersConfig is wrong so it's important that it triggers an error.